backend/src/main/java/com/google/minicrm/servlets/LoginServlet.java

```diff
@@ -54,12 +57,14 @@ public void init() {
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String languageParam = request.getParameter("language");
+    Language language = Language.fromIsoCode(languageParam);
```

Comment:
can languageParam be inlined, so we get Language.fromIsoCode(request.getParameter("language"));? Also, does fromIsoCode handle null/empty case?

Reply:
Yup handles both.